### DISCUSSION TITLE
Miner bootstrap Unable to connect

### DISCUSSION THREAD

**Discussion by lhjok** [filecoin-project/lotus#8196] [Build and Sync] [answered]
lotus-miner net peers No response Miner bootstrap Unable to connect

**Comment by LexLuthr** [comment, marked as answer]
@lhjok Hey, can you explain a bit more about where you are facing the problem exactly? Are you configuring a new miner on local network or calibnet or mainnet? Is this your first attempt at creating a miner? Which document did you follow to setup the miner?

**Reply by lhjok** [reply]
I'm sorry! It's my own problem. My router is not set up properly. It has been solved. Thank you very much for replying to me
